File: Difficulty: Easy/Union of Arrays with Duplicates/union-of-arrays-with-duplicates.cpp
class Solution {
  public:
    vector<int> findUnion(vector<int>& a, vector<int>& b) {
        // code here
        set<int> st;
        
        
        
        for(auto num: a){
            st.insert(num);
        }
        for(auto num: b){
            st.insert(num);
        }
        
        vector<int> ans(st.begin(), st.end());
        
        return ans;
    }
};

File: Difficulty: Easy/Union of Arrays with Duplicates/union-of-arrays-with-duplicates.js
/**
 * @param {number[]} a
 * @param {number[]} b
 * @returns {number[]}
 */
class Solution {
    findUnion(a, b) {
        // code here
        const st = new Set();
        
        for(let num of a){
            st.add(num);
        }
        
        for(let num of b){
            st.add(num);
        }
        
        return Array.from(st).sort((a,b) => a-b);
    }
}